lsp-mode.el

```diff
@@ -1299,12 +1299,19 @@ On other systems, returns path without change."
   "Implemented only to make `company-lsp' happy.
 DELETE when `lsp-mode.el' is deleted.")
 
+(defconst url-path-allowed-chars-without-percent
```

Comment:
We should prefix this with `lsp--` to prevent global namespace collision

Reply:
agree